### ISSUE TITLE
Workflow discussion

### ISSUE THREAD

**Issue by DesislavaMetodieva** [DesislavaMetodieva/meal-master#1]
User Story:
As a developer working on a project, I want to understand the workflow for development within the project so that I can effectively contribute to the codebase and meet project goals.

Acceptance Criteria:
A clear description of the development process should be outlined, including the steps involved and tools used.
The development environment should be set up and described, including required software and dependencies.

1. Create issue in Repo
2. Add user story and acceptance criteria for each task you're working on, even if it's a research task.
3. Assign it to the assignee
4. Start working on the issue

- $ cd project-2 (will change the name when we decide )
- $ git checkout -b html-navbar
- $ git pull origin main
- 
- Work on the task
- Test the code to see if all works as expected
- If all good, push the code to feature branch
- 
- $ git add -A
- $ git commit -m "Add descriptive message"
- $ git push origin html-navbar

5. Ask team mates to review the code before pulling to main branch
6. If all good, push code to main branch by creating a pull request from feature branch to main
7. Delete feature branch after you've completed to work on it
8. Before starting new issue/task, pull the latest file from main
9. Create a local branch following naming convention issue_branch

Repeat all steps from 3 - 9
